taichi/transforms/ir_printer.cpp

```diff
@@ -221,6 +236,7 @@ class IRPrinter : public IRVisitor {
         name = "\"" + std::get<std::string>(c) + "\"";
```

Comment:
I think the ultimate solution is to make use of [std::quoted](https://en.cppreference.com/w/cpp/io/manip/quoted) that can take care of all escape sequences? i.e.:
```suggestion
        name = std::quoted(std::get<std::string>(c));
```
So that it will print `print "Hello, world!\n"` instead of `print "Hello, world!"`, which should be more clear.

Reply:
I tried but `std::quoted` wasn't supposed to be used this way (AFAICT, it should be inserted to a stream). I kind you meant to escape `\n`, so i just added a function for that.